samples/glean/README.md

```diff
@@ -0,0 +1,19 @@
+# [Android Components](../../README.md) > Samples > Glean
+
+![](src/main/res/mipmap-xhdpi/ic_launcher.png)
+
+A simple app showcasing Glean, the mobile telemetry SDK.
```

Comment:
We should link this from the list in the root README file :)

Reply:
Ah! Good point, I knew I was missing something!